massa-bootstrap/src/client_binder.rs

```diff
@@ -115,12 +127,51 @@ impl BootstrapClientBinder {
                 msg
             }
         };
-        Ok(message)
+        Ok((message, Instant::now().duration_since(start)))
```

Comment:
if we manage the timeout in the function i don"t believe

Reply:
I originally added it in because some methods need it, but I'm not sure if all of these need it. This comment was mostly as a reminder for me to double check.